test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Microsoft.OData.Client.TDDUnitTests.csproj

```diff
@@ -81,6 +81,7 @@
     <Compile Include="Tests\Materialization\CollectionValueMaterializationPolicyTests.cs" />
     <Compile Include="Tests\Materialization\EntryValueMaterializationPolicyUnitTests.cs" />
     <Compile Include="Tests\Materialization\CamelCasedTypeMaterializationTests.cs" />
+    <Compile Include="Tests\LoadPropertyTests.cs" />

```

Comment:
shall we put the new test file under "Materialization" subfolder?

Reply:
@xuzhg Done